src/Evolution/Particles/MonteCarlo/Actions/TimeStepActions.hpp

```diff
@@ -131,14 +132,30 @@ struct TimeStepMutator {
     const DirectionalIdMap<Dim, std::optional<DataVector>>&
         cell_light_crossing_time_ghost = mortar_data.cell_light_crossing_time;
 
+    tnsr::iJJ<DataVector, 3, Frame::Inertial> d_inv_spatial_metric =
```

Comment:
call the existing function in PointwiseFunctions/GeneralRelativity/DerivativeSpatialMetric.hpp